### DISCUSSION TITLE
Soak off cap clips

### DISCUSSION THREAD

**Discussion by alejandrosenes** [pdbuzzboard/pdbuzzboard#8] [Ideas]
Hello, thanks for all, great work 
I made my own versión with ethernet conectors and ethernet cat5 flex cable, Soak off cap clips contains the motor.
God luck for all 
![IMG_20230413_184509](https://user-images.githubusercontent.com/22768937/232340120-03e4b57f-0782-47bd-9fdc-e021206932e4.jpg)
![IMG_20230413_184428](https://user-images.githubusercontent.com/22768937/232340121-236fa81d-d594-46c9-8f96-da0e4bcd93c8.jpg)
![IMG_20230413_184515](https://user-images.githubusercontent.com/22768937/232340146-6e8046c1-83fa-423a-ba93-045918bc2168.jpg)

**Comment by PWPInnovator898** [comment]
Looks great!   Can you kindly reference the soak cap clips brand and flex Ethernet cable brand or where to buy?  Thanks!!

**Reply by alejandrosenes** [reply]
Sure! 
Ethernet cable is like this
https://amzn.eu/d/3NHVnva
Soak caps from here
https://a.aliexpress.com/_EH9WQQr
Also avariable in ebay and Amazon... Thanks!

**Comment by FrankLeeCanada** [comment]
Looking good. Thank you for the post.  I am working on Arduino Neno for my wife and waiting for the case to finish my project

**Comment by pdbuzzboard** [comment]
Very nice!  If you are interested, please put together a video (or Powerpoint show if that is easier) of the assembly.  I can put it on the PDBuzzBoard YouTube channel or will put a link to the video where ever you choose to post it.

Congratulations on the build!

**Comment by Domminz** [comment]
Hi everyone,
Kindly share details of soldering the wires. Does the CAT cable have coper wire? the motor connecting wires are copper, as well? or both Alu? How is other end connection to the Arduino board via RJ45 or further wires have to be split?
Thnx.
